.travis.yml

```diff
@@ -84,6 +84,19 @@ matrix:
               packages:
                 - clang-format-9
       env: MATRIX_EVAL="COVERAGE=Off && CLANG_FORMAT=On"
+    - os: linux
+      dist: xenial
+      compiler: clang
+      addons:
+          apt:
+              sources:
+                - ubuntu-toolchain-r-test
+                - llvm-toolchain-xenial-6.0
+                - sourceline: 'deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-9 main'
+                  key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
+              packages:
+                - ['clang-tidy-9', 'clang-9', 'libstdc++-7-dev']
```

Comment:
libstdc++-7-dev or libstdc++-8-dev? Which one is the right one here?

Reply:
As far as I understand, it doesn't need to be a particular version as long as it supports c++17. Note that we never install it for `gcc` builds, which means it uses whatever's already there. I don't remember why we do that for clang, but there might have been some issues with earlier versions, I presume. Note that `libstdc++` is the GNU implementation anyway, not LLVM's.